src/olympia/accounts/views.py

```diff
@@ -163,7 +163,10 @@ def with_user(format):
     def outer(fn):
         @functools.wraps(fn)
         def inner(self, request):
-            data = request.GET if request.method == 'GET' else request.DATA
+            data = (
+                request.query_params
+                if request.method == 'GET'
+                else request.data)
```

Comment:
Matter of personal taste, but IMHO if it needs to be broken in 4 lines, then you might as well just do:
```
if request.method == 'GET':
    data = request.query_params
else:
    data = request.data
```